pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -115,6 +115,7 @@ type nodeagentContext struct {
 	maintModeReason             types.MaintenanceModeReason //reason for entering Maintenance mode
 	configGetSuccess            bool                        // got config from controller success
 	vaultmgrReported            bool                        // got reports from vaultmgr
+	kubeClusterMode             bool                        // image is kubernetes cluster type
```

Comment:
Elsewhere we in EVE-OS we call this either "hv" or "eve-flavor". Let's not invent any new term for it but use one of those.

Reply:
I can change this to eveFlavor here, although I'm thinking about currently we only check on the image file type, later one, we may want to add some configure knobs for decide to run in legacy mode or this new mode with the same image, then it will be more than just the eve image type.

Reply:
It will have to be the same as determined as boot, because the boot time determination affects which EVE-OS containers are started. So you can't decide to use kubevirt independently of that decision.

Reply:
changed to 'hvTypeKube' in this commit.